services-js/access-boston/src/server/services/SamlAuth.ts

```diff
@@ -32,6 +32,7 @@ interface SamlAuthAssertion {
       userAccessToken?: string[];
       userMFARegistrationDate?: string[];
       isUserRegistered?: string[];
+      cobUserAgency?: string[];
```

Comment:
```suggestion
      userCobAgency?: string[];
```

Reply:
^^ just to be consistent with the existing naming (`userAccessToken` et al.)

Reply:
Those names come from the IAM team’s setup, so they’re not in our control.

Reply:
Ah - bummer!